src/loaders/svg/tvgSvgLoader.cpp

```diff
@@ -1397,6 +1442,20 @@ static SvgNode* _findChildById(SvgNode* node, const char* id)
     return nullptr;
 }
 
+static SvgNode* _findNodeById(SvgNode *node, string* id)
+{
+    SvgNode* result = nullptr;
+    if ((node->id != nullptr) && !node->id->compare(*id)) return node;
```

Comment:
coding convention:
if ((node->id) && ...)

Reply:
I add patch to fix this